cob_collision_monitor/launch/collision_monitor.launch

```diff
@@ -1,14 +1,17 @@
 <?xml version="1.0"?>
 <launch>
-  <arg name="load_robot_description" default="false"/>
   <arg name="robot" default="$(env ROBOT)"/>
   <arg name="config_path" default="$(find cob_moveit_config)/$(arg robot)/config"/>
+
+  <arg name="load_semantic_description" default="true"/>
```

Comment:
this should be false if it started by the bringup, i.e. the bringup will take care of it